doc-botonic/docs/concepts/14.humanhandoff.md

```diff
@@ -202,3 +200,74 @@ export default class extends React.Component {
+| `withOnFinishPath` | Perform a handoff and specify a payload or a path to trigger the action that the bot will follow after the conversation with the agent has finished. | path          | string  | Yes  |
+| `withAgentEmail`   | Perform a handoff along with the agent's email.                                                                                                      | email         | string  | Yes  |
+| `withAgentId`      | Perform a handoff along with the agent's ID.                                                                                                         | agentId       | string  | Yes  |
+| `withNote`         | Perform a handoff and leave a message for other agents to know more about the case.                                                                  | note          | string  | Yes  |
+| `withCaseInfo`     | Perform a handoff and give details about the case.                                                                                                   | caseInfo      | string  | Yes  |
+| `withShadowing`    | Perform a handoff while hiding agent's ID and name.                                                                                                  | shadowing     | boolean | No   |
+
+
+
+## Methods to Request Agent's Information
+
+### getOpenQueues
+
+Get the list of all the open queues.
+
+| Property | Type   | Description       |
+|----------|--------|-------------------|
+| queues   | String | Name of the queue |
+
+
+### getAvailableAgentsByQueue
+
+Get the list of all the agents who can attend a case by queue.
+
+| Property | Type   | Description       |
+|----------|--------|-------------------|
+| agents   | String | Name of the agent |
+
+
+### getAvailableAgents
+
+Get extra information from the agents, sorted by the number of cases that are attending in Hubtype Desk. When creating a new case, it enables the case autoassignment to a specific agent.
+
+| Property | Type              | Description                                                                                                          |
+|----------|-------------------|----------------------------------------------------------------------------------------------------------------------|
+| agents   | HubtypeAgentsInfo | Agent's information such as email address, number of cases attending, status (available or not) and last message sent |
+
+**Details**
+
+```javascript
+email: string
+attending_count: number
+status: string
+last_message_sent: string
+```
+
+
+### getAgentVacationRanges
+
+Get the agent's vacation information to know their availability. 
+| Property        | Type          | Description                                      |
+|-----------------|---------------|--------------------------------------------------|
+| agentId         | String        | Agent ID number                                  |
```

Comment:
you're mixing input with output of the function

Reply:
Tables modified